frontend/app/src/App.test.tsx

```diff
@@ -2178,19 +2188,21 @@ describe("App", () => {
 
       sendForwardMessage("newSession", { ...NEW_SESSION_JSON })
 
-      expect(clearInterval).toHaveBeenCalledWith(expect.any(Number))
-      expect(clearInterval).toHaveBeenCalledWith(expect.any(Number))
+      expect(clearInterval).toHaveBeenCalled()
+      expect(clearInterval).toHaveBeenCalled()
     })
 
     it("triggers rerunScript with is_auto_rerun set to true", () => {
       renderApp(getProps())
 
       const connectionManager = getMockConnectionManager()
-      sendForwardMessage("autoRerun", {
-        interval: 1.0,
-        fragmentId: "myFragmentId",
+      act(() => {
+        sendForwardMessage("autoRerun", {
+          interval: 1.0,
+          fragmentId: "myFragmentId",
+        })
+        jest.advanceTimersByTime(1000)
```

Comment:
should jest still be in here?

Reply:
I've aliased jest as vi so it's vi under the hood. Doing a follow up to replace the rest cause it's a drop in replacement. It's basically my attempt to keep this PR as small as possible so you can see the only meaningful changes.

Reply:
oh, nice!